charts/common/values.test.yaml

```diff
@@ -0,0 +1,307 @@
+# start common.ingress
+ingress:
+  deploy: true
+# end common.ingress
+
+# start common.servicemonitor
+servicemonitor:
+  deploy: true
+  basicAuth:
+    enabled: true
+    existingSecret: ""
+    newSecret:
+      username: bedag
+      password: changeit
+  endpoints:
+    komponente:
+      componentSelector:
+        key: app.kubernetes.io/component
+        value: komponente
+# end common.servicemonitor
+
+# start common.pvcs
+pvcs:
+  - name: name-of-pvc
+# end common.pvcs
+
+# components are a dictionary of single components to be configured
+components:
+  component-1:
+    # start common.service
+    services:
+      service-1:
+        deploy: true
+    # end common.service
+    # start common.networkpolicy
+    networkpolicy:
+      deploy: true
+    # end common.networkpolicy
+    controller:
+      deploy: true
+      type: "Deployment"
+
+      containers:
+        container-1:
+          image:
+            repository: registry-group.mgmtbi.ch/docker_base/nginx
```

Comment:
remove internal registry (multiple times in this file)

Reply:
done